src/Screen/VideoPreview/VideoPreview.jsx

```diff
@@ -23,30 +23,35 @@ const VideoPreview = () => {
     watchlistDispatch,
   } = useWatchlist();
 
-  const {
-    historyDispatch
-  } = useHistory();
+  const { historyDispatch } = useHistory();
+
+  const { isModalOpen, setIsModalOpen } = usePlaylist();
 
   useEffect(() => {
     (async () => {
       try {
         const { data } = await axios.get(`/api/video/${id}`);
         setVideo(data.video);
       } catch (error) {
-        alert(error);
+        console.log(error);
       }
     })();
   }, []);
 
   useEffect(() => {
     (async () => {
       try {
-        !isEmptyObject(video) && await historyDispatch({type: 'ADD_TO_HISTORY', payload: video })
+        !isEmptyObject(video) &&
+          (await historyDispatch({ type: "ADD_TO_HISTORY", payload: video }));
       } catch (error) {
-        alert(error);
+        console.log(error);
       }
-    })()
-  }, [video])
+    })();
+  }, [video]);
```

Comment:
You may also check user is logged in  or not

Reply:
For that- I already redirect user to home page even if the user wants to go login page(if loggedIn).